### PR TITLE
Add switchable background music

### PR DESCRIPTION
**Pull request by AdelaCorbeanu**
I added background music in the game by using an Audio Source object and an electronic music asset from the Asset Store (I left its readme file for credit). The music starts when you hit play and it loops endlessly. You can switch to another track by pressing Ctrl (it works with both Left Ctrl and Right Ctrl). There are currently 6 tracks that you can choose from. When you die, the music stops, and if you click replay it starts from the track that was playing when you died. 

I had to enable this for each music track so that the transition between music tracks doesn't cause lag:
image
It didn't seem to affect anything.

### Future: 
I'm thinking of also adding events for the keys Plus (+) and Minus (-) to adjust the volume (so that we don't create an entire Settings page just for this)

### CODE REVIEW

File: Flight Game/Assets/Scripts/Game/Music.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Music : MonoBehaviour
+{
+    public AudioClip[] backgroundMusicClips;
+    private static int currentClipIndex = 0;
```
Comment (AdelaCorbeanu) [inline Flight Game/Assets/Scripts/Game/Music.cs:8]: This needs to include the private naming convention

File: Flight Game/Assets/Scripts/Game/Music.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    private void Start()
+    {
+        SetBackgroundMusic(currentClipIndex);
+    }
+
+    private void Update()
+    {
+        // Check if the CTRL key is released (key up event)
+        if (Input.GetKeyUp(KeyCode.LeftControl) || Input.GetKeyUp(KeyCode.RightControl))
```
Comment (widdrr) [inline Flight Game/Assets/Scripts/Game/Music.cs:18]: Ideally we'd use the Input System for this

**Review by widdrr — APPROVED** [review]
I strongly suggest we do volume controls as soon as possible. Audio without a way to turn it down/up is a great way to annoy players.

**Comment by widdrr** [comment]
I'm also wondering if we can do anything about the filesize.
The added tracks amount to ~320 KB, which is 90% of our entire Assets folder

**Pull request merged**